src/esbmc/options.cpp

```diff
@@ -200,7 +200,8 @@ const struct group_opt_templ all_cmd_options[] = {
     {"no-slice", NULL, "do not remove unused equations"},
     {"multi-fail-fast",
      boost::program_options::value<int>()->value_name("n"),
-     "stops after first n VCC violation found in multi property mode"},
+     "stops after first n VCC violation found in multi property mode."
+     "(not compatible with incremental bmc strategies)"},
```

Comment:
What does the plural form "incremental bmc strategies" refer to? Just `--incremental-bmc` or also k-induction or anything else? Please clarify.

Reply:
It Should at least include `incremental-bmc`/`k-induction`/`k-induction-parallel`. 

Will change the description